### DISCUSSION TITLE
阿盘出错，导致连后台都无法进。bug可以复现。

### DISCUSSION THREAD

**Discussion by barnett2010** [AlistGo/alist#7189] [Q&A]
我是win。正常启动alist.exe。326版本。一直是正常用的。
然后今天狂弹这个提示。下方有日志。
api.xhofe.top/alist/ali_open/token 说阿盘有问题
然后首页提示。获取设置失败： 请稍候，正在加载储存
后台也进不去
http://192.168.1.2:5244/@manage/storages/edit/1
因为点后台也是这行提示，压根不显示菜单了。

于是我在另一台电脑上，从头操作一次。
新建盘，增加阿盘，点保存，又是弹下面的提示，
然后刷新一下网页，提示，获取设置失败： 请稍候，正在加载储存
后台也打不开了。

阿盘app我可以正常使用。
也搜索了相关，但不知如何解决。来求助各位大佬。

`alist server
INFO[2024-09-16 14:27:34] reading config file: data\config.json
INFO[2024-09-16 14:27:34] load config from env with prefix: ALIST_
INFO[2024-09-16 14:27:34] init logrus...
INFO[2024-09-16 14:27:34] start HTTP server @ 0.0.0.0:5244
INFO[2024-09-16 14:27:34] qbittorrent not ready.
INFO[2024-09-16 14:27:34] Aria2 not ready.
2024/09/16 14:27:55.935657 ERROR RESTY Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 1
2024/09/16 14:28:17.044628 ERROR RESTY Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 2
2024/09/16 14:28:38.184065 ERROR RESTY Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 3
2024/09/16 14:28:59.590192 ERROR RESTY Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 4`

**Comment by barnett2010** [comment]
之后一直等，它不重试了。
刷新，后台出来了。

> 2024/09/16 14:38:49.630814 ERROR RESTY Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 4
ERRO[2024-09-16 14:38:49] failed get enabled storages: Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
failed init storage
github.com/alist-org/alist/v3/internal/op.initStorage
        /source/internal/op/storage.go:99
github.com/alist-org/alist/v3/internal/op.LoadStorage
        /source/internal/op/storage.go:80
github.com/alist-org/alist/v3/internal/bootstrap.LoadStorages.func1
        /source/internal/bootstrap/storage.go:20
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1650


点进存储，看到阿盘提示。

> 
阿里云盘OPEN
状态: 

Post "https://api.xhofe.top/alist/ali_open/token": dial tcp 27.151.28.92:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

是不是阿盘不行了？

**Comment by barnett2010** [comment]
我在令牌获取页面
https://alist.nn.ci/tool/aliyundrive/request.html
切换nnci。是上面的错误。
切换top，点scan，出不来二维码，直接提示error

**Comment by AspirantH** [comment]
还以为是我一个人遇到的问题

**Reply by liyujian-prc** [reply]
我也遇到同样问题！

**Reply by chrislxn** [reply]
同样问题+1

**Comment by chrislxn** [comment]
是亚洲云的API挂了，切换到https://api-cf.nn.ci/恢复正常
#7187
